### PR TITLE
Add Raymarching Glass Refraction Examples

### PR DESCRIPTION
**Pull request by ammein**
Refer to the new lygia feature added in this PR https://github.com/patriciogonzalezvivo/lygia/pull/67 🥳

### CODE REVIEW

**Comment by patriciogonzalezvivo** [comment]
This is great! Thanks so much. Do you mind removing those extra files from the commit?

**Comment by ammein** [comment]
> This is great! Thanks so much. Do you mind removing those extra files from the commit?

Mind pointing me to which files need to remove from the commit? 😁

File: filter_bilateralBlur2D.frag
[the remark this answers is not in the document]
Comment (ammein) [reply filter_bilateralBlur2D.frag:1]: Done!

File: space_displace.frag
[the remark this answers is not in the document]
Comment (ammein) [reply space_displace.frag:1]: Done!

**Comment by ammein** [comment]
Hey @patriciogonzalezvivo , the files to remove from the commit have been successfully deleted. Did some minor changes on raymarching glass `defines` refer to this PR https://github.com/patriciogonzalezvivo/lygia/pull/68 . 

Let me know if you need anything else. Can't wait to have this live! 🚀

**Pull request merged**